benchmark/fs/bench-stat-promise.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+
+const bench = common.createBenchmark(main, {
+  n: [20e4],
```

Comment:
This could just be 2e5?

Reply:
Yeah. There are a couple like this in the benchmark suite. I'll update those all separately.